camelComponent/src/main/java/io/vantiq/extsrc/camel/VantiqEndpoint.java

```diff
@@ -71,6 +71,11 @@ public class VantiqEndpoint extends DefaultEndpoint {
     @Setter
     private boolean noSsl;
     
+    @UriParam(defaultValue = "false")
```

Comment:
This sets up the flag, which is ignored by everyone except the VantiqConsumer code